tests/archive/tests_management_command.py

```diff
@@ -255,6 +255,37 @@ def _get_anonymized_jobseeker(self):
             )
         )
 
+    def _get_anonymized_application(self):
+        return list(
+            AnonymizedApplication.objects.all().values(
+                "job_seeker_birth_year",
+                "job_seeker_department_same_as_company_department",
+                "sender_kind",
+                "sender_company_kind",
+                "sender_prescriber_organization_kind",
+                "sender_prescriber_organization_authorization_status",
+                "company_kind",
+                "company_department",
+                "company_naf",
+                "company_has_convention",
+                "applied_at",
+                "processed_at",
+                "last_transition_at",
+                "had_resume",
+                "origin",
+                "state",
+                "refusal_reason",
+                "has_been_transferred",
+                "number_of_jobs_applied_for",
+                "has_diagoriente_invitation",
+                "hiring_rome",
+                "hiring_contract_type",
+                "hiring_contract_nature",
+                "hiring_start_date",
+                "has_approval",
+            )
+        )
```

Comment:
le problème à énumérer les champs c'est qu'on risque d'oublier d'ajouter les nouveaux champs si on fait un changement.
L'avantage c'est que tu peux ne pas passer de valeurs dans `values()` et récupérer tout :)
(valable pour les autres tests où tu fais ça)


```suggestion
        return list(AnonymizedApplication.objects.all().values())
```